package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chpr-worker",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
please use 2.2 for the hotfix https://github.com/ChauffeurPrive/stakhanov/commits/0.2.2 :)

Reply:
done